src/containers/education/Education.css

```diff
@@ -0,0 +1,46 @@
+.education-heading {
+  font-size: 56px;
```

Comment:
Sorry I misclicked "Resolve Conversation" on your comment for this code block. Yes I set it to 56px to be consistent with other sections. For example, `Skills.css` has a heading of 56px on large screens, 40px on medium screens, and 30px on small screens. I made this education heading the same way.

Reply:
@kartikcho

Reply:
Ah I see, it's alright then. 
Also, you can unresolve a comment if you accidentally clicked on it so don't worry about it. As a general good practice you should mark the comments addressed in a follow up commit as resolved.